### ISSUE TITLE
syncthing cli --help doesn't work without a valid home path with cert.pem

### ISSUE THREAD

**Issue by tomasz1986** [syncthing/syncthing#7588]
1. Try running `syncthing cli --help` without starting Syncthing at least once first, or when using a custom home/config folder.
2. The command fails with the following error.
   ```
   reading device ID: open C:\Users\User\AppData\Local\Syncthing\cert.pem: The system cannot find the path specified.
   ```

Also, not sure if this should be a different issue or not, but if I try running

```
syncthing cli --home=<path-to-home> --help
```

then I only get

```
getting config: unexpected HTTP status returned: 401 Unauthorized
Not Authorized
```
.

**Comment by calmh** [comment]
So the thing here is that the CLI code actually talks to Syncthing in order to build the `--help` output, because it lists the config commands and such which depend on the actual config returned by Syncthing. Though if we can't talk to Syncthing to build the config we could show some general options and explain the situation.

**Comment by AudriusButkevicius** [comment]
I don't think it needs to do that? Or atleast didn't need it before?

**Comment by calmh** [comment]
Well I can't speak to the necessity or the history, but that appears to be what it does now.

**Comment by tomasz1986** [comment]
> Though if we can't talk to Syncthing to build the config we could show some general options and explain the situation.

What to do about the `401 Unauthorized` error?

The configuration that I tried running the `syncthing cli --home=<path-to-home> --help` command on is just my test instance of Syncthing with no HTTPS enabled, and also no user and no password set.

**Comment by imsodin** [comment]
@tomasz1986 This PR should fix your issues: https://github.com/syncthing/syncthing/pull/7591  
I don't know where the 401 comes from - if that persists with the PR lets debug that on the forum first.

**Comment by AudriusButkevicius** [comment]
Had a look again, and we do indeed need the config to get the list of devices, as when we construct the command line, we create commands for managing devices. :(

Making some comments on the PR on how I think this could be worked around.

**Comment by imsodin** [comment]
Yes, that was fixed by https://github.com/syncthing/syncthing/pull/7591 which was missing a `#` in the commit message - thanks for noticing.
